components/theme/app/AsideBottom.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <div>
-    <a
+    <AppLink
       v-if="
         masterCoursesLink &&
         ($docus.currentPath.value.startsWith('/docs') || $docus.currentPath.value.startsWith('/examples'))
       "
       :href="masterCoursesLink.href"
-      rel="noopener nofollow noreferrer"
+      rel="noopener"
       target="_blank"
```

Comment:
You can remove the `target` prop now, no longer used.

Reply:
Sure, removed @smarroufin